packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -561,11 +561,14 @@ export default {
             state.orderId = orderId;
         },
 
-        [UPDATE_GEO_LOCATION]: (state, [lng, lat]) => {
-            state.geolocation = {
-                latitude: lat,
-                longitude: lng
-            };
+        [UPDATE_GEO_LOCATION]: (state, coords) => {
```

Comment:
Might be worth writing some unit tests to verify this new behaviour?

Reply:
There is one test to see if the state is not updated if the coordinates are not an array which I assume covers it?